copyright_header/copyright_header.py

```diff
@@ -0,0 +1,221 @@
+# (C) Copyright 2005-2020 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in enthought/LICENSE.txt and may be redistributed only
+# under the conditions described in the aforementioned license.  The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+
+import re
+import tokenize
+
+# Minimum end year for the copyright statement.
+MINIMUM_END_YEAR = 2020
+
+# Regular expression to match things of the form "2019" or of the form
+# "2010-2019".
+YEAR_RANGE = r"(?P<start_year>\d{4})(?:\-(?P<end_year>\d{4}))?"
+
+# Generic copyright, used for searching for multiple copyright headers.
+GENERIC_COPYRIGHT = re.compile("# .*Copyright .*Enthought", re.IGNORECASE)
+
+# Template for a regular expression for the copyright header.
+PRODUCT_CODE_HEADER_TEMPLATE = r"""
+# \(C\) Copyright {year_range} {company_name}
+# All rights reserved\.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in enthought/LICENSE\.txt and may be redistributed only
```

Comment:
`enthought/LICENSE.txt` should probably just be `LICENSE.txt` (I realise that's not what is in most files, so we might want to fix that in a separate PR).

Reply:
Right, fixing the actual headers will be a separate PR; every single file will need changes.